### ISSUE TITLE
After upgrade from Craft 3 a form content column was dropped

### ISSUE THREAD

**Issue by gglnx** [verbb/formie#1824]
### Describe the bug

After upgrade my Craft 3 installation to Craft 4 one of my form content columns with content will be dropped while migration:

```
*** applying m221206_000000_fix_syncs
> drop column field_young_beitrag from table {{%fmc_foerdermitgliedschaft}} ... done (time: 0.004s)
    > Dropped column field_young_beitrag in {{%fmc_foerdermitgliedschaft}}.
```

I can not change the field handle, because it's used in an integration.

### Steps to reproduce

1. Create a form with a field with `young_beitrag` (without suffix) in a Craft 3 installation
2. Try to upgrade

### Form settings

- Multi-page form: Yes
- Submission Method: Page Reload
- Client-side Validation: No
- Custom Form Templates: Yes

### Craft CMS version

4.8.9

### Plugin version

2.1.10

### Multi-site?

Yes

**Comment by engram-design** [comment]
This would from the [sync fixing migration](https://github.com/verbb/formie/blob/9f0d32891e5ec9eae2341b2c33cb814a993e58f7/src/migrations/m221206_000000_fix_syncs.php#L57-L59) where we try and determine the content column for the field. Having an underscore in the field handle and no suffix (this field probably was created on an older Craft 3 install) means that it's tricky to figure out of `beitrag` is part of the field handle or just the suffix. Sometimes even on newer Craft 3 installs, the suffix is missing.

Fixed for the next release. To get this early, run `composer require verbb/formie:"dev-craft-4 as 2.1.10"`.

**Comment by engram-design** [comment]
Fixed in [2.1.11](https://github.com/verbb/formie/releases/tag/2.1.11)
